tests/conftest.py

```diff
@@ -70,9 +76,13 @@ def init_test_schema(request):
     This is automatically called per test session.
     """
     param = get_cnx_param()
+    TEST_DB = param['database']
     with snowflake.connector.connect(**param) as con:
+        con.cursor().execute("CREATE OR REPLACE DATABASE {0}".format(TEST_DB))
```

Comment:
This would fail on travis tests because you won't be able to have the privilege to create database.

Reply:
Removed the creation of Database. Resolving this conversation since the tests passed. Will squash the commits once approved.